### ISSUE TITLE
Bug with 'new match'

### ISSUE THREAD

**Issue by mayeaux** [tinderjs/tinder-desktop#135]
Even if it's not a new match, my latest match is alerted to me everytime I boot the app. The logic that checks if the match is new based on lastActivityDate is bugged

**Comment by SaturnusDJ** [comment]
Happens over here after resign in via Fb. Last match always notified about.

**Comment by mayeaux** [comment]
Yeah, so I'm pretty sure that there is a concept of 'lastActivityDate', which is obviously not being updated or something like that, we'll see, relatively low in priority scale but it's kind of annoying.
